src/server/ocpp/utils/OCPPUtils.ts

```diff
@@ -986,7 +986,7 @@ export default class OCPPUtils {
     return foundTemplate;
   }
 
-  public static async enrichChargingStationWithTemplate(tenantID: string, chargingStation: ChargingStation): Promise<TemplateUpdateResult> {
+  public static async enrichChargingStationWithTemplate(tenantID: string, chargingStation: ChargingStation, manualCall?: boolean): Promise<TemplateUpdateResult> {
```

Comment:
Why the manualCall variable here?
The manualConfiguration is part of the charging station object

Reply:
@LucasBrazi06  
The manual call is needed, because we need to call `enrichChargingStationWithTemplate`  after the manual configuration is deactivated. 
If we would call the method without the extra parameter, the if statement `(chargingStation.templateHash !== chargingStationTemplate.hash)` would be false as there is still a valid template hash in the station (Only the technical one is deleted). So the charging station would be not enriched with the technical details of the template.

It is implemented this with the intention to still update all the other hashes like capabilities etc., even if manual config is activated.

Reply:
There should not be a call to this method after manual configuration is done.
This method is called only at BootNotif time and by a scheduled job.